python_modules/dagster/dagster/_cli/asset.py

```diff
@@ -111,7 +112,46 @@ def asset_list_command(**kwargs):
     asset_keys = asset_selection.resolve(repo_def.asset_graph, allow_missing=True)
 
     for asset_key in sorted(asset_keys):
-        print(asset_key.to_user_string())  # noqa: T201
+        print(asset_key.to_user_string())
+
+
+@asset_cli.command(name="report", help="Report asset materialization")
```

Comment:
Thoughts on calling this `report-materialization`? Thinking about leaving room for other kinds of events we might want to support reporting in the future.